NSS-Day-Cohort-53/kennels-econolines
# Added Total Employees and Total Animals to displayed locations

#### Changes Made
1. Modified file `location.js` to include `employeeLocations and Animals` 
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout sh-getLocations
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > When locations page loads total number of animals and employees will be displayed on each location card
    > Click locations and both totals will display.
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
